std_msgs/Header header

uint8 index
int32 mot1_pwm
int32 mot2_pwm